lib/services/podcast-index/fetch-episodes.ts

```diff
@@ -0,0 +1,42 @@
+import { podcastIndexFetchClient } from './client';
+
+export type PodcastIndexEpisodeType = {
+  datePublished: number;
+  datePublishedPretty: string;
+  description: string;
+  duration: number;
+  enclosureUrl: string;
+  episode: number;
+  feedImage: string;
+  feedLanguage: string;
+  guid: string;
+  image: string;
+  language: string;
+  size: number;
+  title: string;
+};
+
+type PodcastIndexResponse = {
+  count: number;
+  items: PodcastIndexEpisodeType[];
+  status: boolean;
+};
+
+export const fetchEpisodes = async (
+  podcastGuid: string,
+): Promise<PodcastIndexEpisodeType[]> => {
+  const response = await podcastIndexFetchClient<PodcastIndexResponse>(
+    `/episodes/bypodcastguid?guid=${podcastGuid}`,
```

Comment:
**nit:** we may use `params` instead of appending `?guid=` to the URL to keep it cleaner.